common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -234,6 +236,13 @@ class LTIFields(object):
         scope=Scope.settings
     )
 
+    accept_grades_past_due = Boolean(
+        display_name=_("Accept grades past deadline"),
+        help=_("Select True to allow third party systems to post grades past the deadline."),
+        default=False,
```

Comment:
Since currently there is no check for due dates when accepting grades, settings this to `False` will change the behaviour for existing modules. So shouldn't this be `True` by default?